docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide covers mLRS in its core use case to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use OpenTX or EdgeTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
```

Comment:
I'm not happy with calling this its core use case.
maybe "... in one of its core use cases to ..."

"... use EdgeTx or OpenTx ..."

Reply:
IMO, this is still the core use case and also the most popular, but I think I'll remove reference to "core" here entirely.

Reply:
I would prefer that